examples/with-nextjs/frontend/src/pages/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Inter } from "next/font/google";
+import { formatEther } from "viem";
+import CountUp from "react-countup";
+
+import { useDeposits, type Deposit } from "../hooks/useDeposits";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
+  const deposits = useDeposits();
+
+  return (
+    <main
+      className={`flex min-h-screen flex-col items-center justify-between pt-24 ${inter.className}`}
+    >
+      <div className="w-full max-w-2xl flex flex-col p-4 gap-6">
+        <h1 className="font-bold text-2xl">10 latest WETH mints</h1>
+
+        <div className="w-full flex gap-1 flex-col justify-between items-center">
+          {deposits.status === "pending" ? (
+            <p className="font-semibold">Loading...</p>
+          ) : deposits.status === "error" ? (
+            <p className="font-semibold text-red-500">Error fetching mints</p>
+          ) : (
+            <Table deposits={deposits.data} />
+          )}
+        </div>
+      </div>
+    </main>
+  );
+}
+
+function Table({ deposits }: { deposits: Deposit[] }) {
```

Comment:
Removed the `_document.tsx` file and moved the Table component inline, just the reduce the number of files folks need to browse to the gist of things.